trainee_template/conviction.amlg

```diff
@@ -705,245 +705,6 @@
-	)
-
-	;helper method for PredictionResidualConviction to compute, cache and output the expected global residual vector magnitude
-	#!ComputeExpectedResidualVectorMagnitude
-	(seq
-		(declare (assoc
-			k_parameter (get hyperparam_map "k")
-			p_parameter (get hyperparam_map "p")
-			dt_parameter (get hyperparam_map "dt")
-			feature_deviations (if (get hyperparam_map "useDeviations") (get hyperparam_map "featureDeviations"))
-			query_feature_attributes_map (get hyperparam_map "featureDomainAttributes")
-		))
-
-		(declare (assoc
-			model_predicted_case_residuals
-				;iterate over all cases and compute each cases's predicted residuals for action_features
-				(map
-					(lambda (let
-						(assoc
-							case_id (target_value 1)
-							individual_case_residuals_map (assoc)
-							local_cases_map (assoc)
-						)
-						(assign (assoc
-							local_cases_map
-								(compute_on_contained_entities (list
-									(query_not_in_entity_list (list case_id))
-									(query_nearest_generalized_distance
-										k_parameter
-										features
-										(retrieve_from_entity case_id features)
-										feature_weights
-										queryDistanceTypeMap
-										query_feature_attributes_map
-										feature_deviations
-										p_parameter
-										dt_parameter
-										(if valid_weight_feature weight_feature (null))
-										(rand)
-										(null) ;radius
-										numericalPrecision
-									)
-								))
-						))
-
-						(assign (assoc
-							individual_case_residuals_map
-								(get
-									(call ExpandResidualValuesToUncertainty  (assoc
-										feature_residuals_map
-											(call CalculateFeatureResiduals (assoc
-												features features
-												target_residual_feature (first action_features)
-												cases_map local_cases_map
-												use_case_weights use_case_weights
-												weight_feature weight_feature
-												custom_hyperparam_map hyperparam_map
-												focal_case case_id
-											))
-									))
-									"residual_map"
-								)
-						))
-						(declare (assoc action_residuals (unzip individual_case_residuals_map action_features)))
-
-						(if (> (size action_features) 1)
-							(generalized_distance
-								;weights are null for all action features
-								(null)
-								;null types and limits so that all the feature residual values are treated as continuous
-								(null)
-								(null)
-								(unzip feature_deviations action_features)
-								p_parameter
-								action_residuals
-								(null) ;null point
-								(null)
-								action_features
-							)
-							;else
-							(first action_residuals)
-						)
-					))
-					(call AllCases)
-				)
-		))
-
-		(declare (assoc
-			expected_residual_vector_magnitude (/ (apply "+" model_predicted_case_residuals) (size model_predicted_case_residuals))
-		))
-
-		;store into residualVectorMagnitudeMap
-		(assign_to_entities (assoc
-			residualVectorMagnitudeMap
```

Comment:
this is a model parameter, check if it's used anywhere else, if not should be removed from trainee template.

Reply:
ah right. Doesn't seem to be anywhere else, removing now.